apollo-compiler/src/test/graphql/com/example/input_object_type/ColorInputExpected.java

```diff
@@ -0,0 +1,57 @@
+package com.example.input_object_type;
+
+import java.lang.Double;
+import javax.annotation.Nullable;
+
+public final class ColorInput {
+  int red = 1;
+
+  @Nullable Double green = 0.0;
+
+  double blue = 1.5;
```

Comment:
Instead of assigning the fields directly, the builder should initialize them in a constructor